datahub-web-react/src/app/ingest/source/builder/RecipeForm/FormField.tsx

```diff
@@ -88,22 +89,26 @@ function ListField({ field, removeMargin }: ListFieldProps) {
     );
 }
 
-function SelectField({ field }: SelectFieldProps) {
+function SelectField({ field, removeMargin }: SelectFieldProps) {
     return (
-        <Form.Item
+        <StyledFormItem
             name={field.name}
             label={field.label}
             tooltip={field.tooltip}
             style={{ flexDirection: 'row', width: '80%', display: 'flex', alignItems: 'baseline' }}
+            removeMargin={!!removeMargin}
         >
             {field.options && (
-                <StyledSelectField>
+                <StyledSelectField
+                    placeholder={field?.placeholder}
+                    defaultValue={field && field.options && field.options[0]}
```

Comment:
unfortunately setting the `defaultValue` on fields in this form won't really help us out since we're relying on Ant design's form state - all initial values are going to be set on the parent `Form` component when we pass in `initialValues` - and those are derived from the `placeholderRecipe` string that we define for each ingestion source (in the folder `src/app/ingest/source/conf`)

Reply:
Done!